### PR TITLE
chrore(trestle): Repo initialisation

### PR DESCRIPTION
**Pull request by aNebula**
Setup commit and CI pipeline, with minimal project structure.

### CODE REVIEW

**Review by brunomarq — APPROVED** [review]
Looks good to me.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Software License 2.0. Using the SPDX format is the simplest approach.
+e.g.
+
+```
+/*
+Copyright (c) 2020 IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+```
```
Comment (butler54) [inline CONTRIBUTING.md:47]: Lets put this in the same style we have used on the project.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+Signed-off-by: John Doe <john.doe@example.com>
+```
+
+You can include this automatically when you commit a change to your
+local git repository using the following command:
+
+```
+git commit -s
+```
+
```
Comment (butler54) [inline CONTRIBUTING.md:70]: Let's make sure the DCO.txt is in the project root.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Christopher Butler - chris.butler@au1.ibm.com
+
```
Comment (butler54) [inline MAINTAINERS.md:2]: Lets put people's github handles rather than emails.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+To contribute code or documentation, please submit a [pull request](https://github.com/IBM/compliance-trestle/pulls).
+
+A good way to familiarize yourself with the codebase and contribution process is
+to look for and tackle low-hanging fruit in the [issue tracker](https://github.com/IBM/compliance-trestle/issues).
+Before embarking on a more ambitious contribution, please quickly [get in touch](MAINTAINERS.md) with us.
+
+**Note: We appreciate your effort, and want to avoid a situation where a contribution
+requires extensive rework (by you or by us), sits in backlog for a long time, or
+cannot be accepted at all!**
+
```
Comment (butler54) [inline CONTRIBUTING.md:14]: Reference the code of conduct. See the auditree contributing file(complianceascode/auditree-framework).

File: README.md [hunk 5]
```diff
@@ -1,2 +1,81 @@
...
+### Machine readable compliance format
+
+Compliance activities at scale, be that size of estate, or number of accreditations, require automation to be successful & repeatable. OSCAL as a standard allows teams to bridge between the "Governance" layer and operational tools.
+
+By building human managed artifacts into OSCAL, Trestle is not only able to validate the integrity of the artifacts that people generate, it also enables reuse and sharing of artifacts and can also provide suitable input into tools which automate operational compliance.
+
+## Using Trestle
+
+Trestle converts complex schema/data structures into simple files in a directory structure. The aim of this is to make it easier to manage for humans - individual objects can be versioned & reviewed, then 'compiled' into the larger structure of a Catalog, SSP or Assessment Plan.
+
```
Comment (butler54) [inline README.md:28]: adding issue #16 which I will cover after merged.

**Review by butler54 — CHANGES_REQUESTED** [review]
Small changes as discussed. Is looking good.

**Review by butler54 — APPROVED** [review]

**Pull request merged**